strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFileTests/Tests.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Resources;
+
+namespace Tests
+{
+    [TestClass]
+    public class Tests 
+    {
+        const string HELLO_TEXT = "Hello, how are you?";
+        readonly ResourceManager _rmEnglish = new ResourceManager(@"ReadStringFromResourceFile.Resources.Texts.English", typeof(ReadStringFromResourceFile.Resources.Texts.English).Assembly);
+
+        [TestMethod]
+        public void GivenEnglishResourceFileHandler_WhenRunIsCalled_ThenResourceStringFetchedIsCorrect()
```

Comment:
Why do we have the `WhenRunIsCalled` part in the test method name? Where are we calling `Run`?